### DISCUSSION TITLE
Pin spotlighted lobbies in the playlists listing

### DISCUSSION THREAD

**Discussion by venix12** [ppy/osu#16936] [Ideas] [answered]
Currently, spotlighted lobbies are often getting lost in the crowd of newly created ones, think it would make sense to have spotlighted lobbies always on the top of listing, that could make them more visible as sort of "officially featured".

**Comment by peppy** [comment, marked as answer]
Can agree with this. I think this was the eventual goal, but we can probably make it happen client-side with the way the listing is being returned. I'm currently doing some maintenance around this to make it faster, so will look at making this change as well.

**Reply by peppy** [reply]
#16943
